eradiate/radprops/particle_dist.py

```diff
@@ -58,68 +78,58 @@ class VerticalDistribution(ABC):
         type="float",
     )
 
-    def __attrs_post_init__(self):
-        if self.bottom >= self.top:
+    @bottom.validator
+    @top.validator
+    def _bottom_and_top_validator(instance, attribute, value):
+        if instance.bottom > instance.top:
```

Comment:
Won't this break if `instance.bottom == instance.top`?

Reply:
Oops, I forgot to correct that, you're right, thank you!